lamindb_setup/core/_settings_storage.py

```diff
@@ -170,7 +170,11 @@ def init_storage(
     # _select_storage inside init_storage_hub also populates ssettings._uuid
     # and we don't want to delete an existing storage record here if no write access
     # only newly created
-    if hub_record_status == "hub-record-created" or not ssettings.type_is_cloud:
+    # local storages not registered in the hub should be also marked
+    is_local_not_retrieved = not (
+        ssettings.type_is_cloud or hub_record_status == "hub-record-retireved"
```

Comment:
There is a typo in the return code! ☺️ -- That has to be fixed in another PR.

Reply:
Oh, and it is even bigger that than that, but at least it was consistent, so actually nothing got broken due to the typos. https://github.com/laminlabs/lamindb-setup/pull/1001